Assets/Scripts/ProximityTrigger.cs

```diff
@@ -112,6 +113,7 @@ void ActivateTrigger()
 	void Active()
 	{
 		active = true;
+		StartCoroutine( MonitorTriggerPlayerDist() );
```

Comment:
Starts any coroutine as soon as it becomes active from an event (e.g. DadDoor's trigger on HasDadKey Event)